### ISSUE TITLE
tns test command never ends in 2.5 and breaks CI builds

### ISSUE THREAD

**Issue by PeterStaev** [NativeScript/nativescript-cli#2505]
I have been using `tns test <platform> --emulator --justlaunch` to configure CI in Travis. Pre-2.5 once the tests were completed the commands exits thus enabling the TravisCI script to continue with the remaining commands and complete the build. After upgrading to 2.5 this is no longer the case. The `tns test` command just stalls after the tests are completed (application is closed on the emulator) and the TravisCI build hangs and timeouts. You can see the problem [here](https://travis-ci.org/PeterStaev/nativescript-azure-mobile-apps/builds/199314626#L3276) also verified locally the same problem happens on both iOS and Android.

**Comment by mauriciomelo** [comment]
Same issue here.

**Comment by mauriciomelo** [comment]
I have a "workaround" that works here, steps:
### set singleRun to true on  karma.conf.js
```

   [...]
    // Continuous Integration mode
    // if true, Karma captures browsers, runs the tests and exits
    singleRun: true
```

### and run

 
 ```tns test <platform> --emulator --justlaunch```

**Comment by PeterStaev** [comment]
Thanks @mauriciomelo ! Your workaround worked excellent here too!

**Comment by dtopuzov** [comment]
Fixed with {N} 3.0.0.
